rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -40,3 +51,55 @@ u64 _sys_game_get_system_sw_version()
 {
 	return stof(utils::get_firmware_version()) * 10000;
 }
+
+error_code _sys_game_watchdog_start(u32 time)
+{
+	sys_game.trace("sys_game_watchdog_start(time=%d)", time);
+
+	if (!watchdog_stopped)
+	{
+		return CELL_EABORT;
+	}
+
+	auto watchdog = [=]()
+	{
+		while (!watchdog_stopped)
+		{
+			if (get_timestamp() - watchdog_last_clear > time * 1000 * 1000)
```

Comment:
I think "time" should be called "timeout". Preferable with the proper unit as suffix in the name.

Reply:
Updated.